spec/lib/cap_authors_poller_spec.rb

```diff
@@ -1,4 +1,3 @@
-
 describe CapAuthorsPoller, :vcr do
   # The author is defined in /spec/factories/author.rb
   let(:author) { create :author }
```

Comment:
Can this be changed to use `build` instead?  The `build` option does not save it.
- https://devhints.io/factory_bot

Reply:
Maybe!  I added both database and model constraints on uniqueness though, so I think testing "through" the database is fine for the scope of this PR.  I don't think that refactoring of the spec is tied to cap_profile_id uniqueness.

Reply:
Wondering whether using a `build` would allow the other mocks to `save` and call original.